### PR TITLE
Release/v0.2.0

### PR DESCRIPTION
**Pull request by No3ml3**
## Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [ ] Frontend
- [ ] Backend
- [ ] Other: .....

## General checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I ran `bun run format`/`bun run lint` before commit
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation where needed
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings
- [ ] I have updated `develop` and merged to my branch before submitting pull request
- [ ] My pull request generate no conflicts with `develop` branch
- [ ] I requested code review from other team members

## Frontend checklist:

- [ ] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [ ] My `Javascript` generate no new console errors
- [ ] I tested my code cross browsers
- [ ] My slice is pixel perfect for both desktop and mobile according to design
- [ ] I conducted basic QA to assure all features are working
- [ ] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ ] I tested admin by manually adding content from zero
- [ ] I created easy to use admin experience which is self-explanatory
- [ ] I added description to admin fields in hard-to-understand areas
- [ ] I followed guidelines for naming `Typescript` variables
- [ ] I conducted basic QA to assure all features are working

### CODE REVIEW

**Review by jaalazard — DISMISSED** [review]

**Review by Justicesvr — DISMISSED** [review]

File: packages/frontend/web/src/components/CardImage.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  readonly image: string | undefined;
+  readonly totalDegree: number;
+}
+const picture = (image: string | undefined, total_degree: number) => {
+  if (image === null) {
+    return total_degree > 0
+      ? '/placeholder-cocktail.webp'
+      : '/placeholder-cocktail-virgin.webp';
+  } else {
+    return `${import.meta.env.VITE_BACKEND_URL}/${image}`;
```
Comment (YDrogen) [inline packages/frontend/web/src/components/CardImage.tsx]: Mauvaise URL

**Review by YDrogen — CHANGES_REQUESTED** [review]

File: packages/frontend/web/src/components/CardImage.tsx (hunk 1 above)
Comment (No3ml3) [reply packages/frontend/web/src/components/CardImage.tsx]: comprends pas c quoi mauvaise url

Comment (YDrogen) [reply packages/frontend/web/src/components/CardImage.tsx]: Le lien n'est pas bon, ça ne fonctionnera pas tel quel

Comment (No3ml3) [reply packages/frontend/web/src/components/CardImage.tsx]: ca focntionne sur localhost ou alors il faut que je rajoute /uploads car le ${import.meta.env.VITE_BACKEND_URL} ne comprends pas uploads ????

Comment (YDrogen) [reply packages/frontend/web/src/components/CardImage.tsx]: Comme on a fait avec le déploiement pour remplacer la variable d'env par `/api`, il faut faire pareil avec `/uploads`

**Review by Justicesvr — DISMISSED** [review]

**Review by jaalazard — APPROVED** [review]

**Review by Justicesvr — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Review by Cardamced — APPROVED** [review]

**Pull request merged**